Add SASL description and config options (#3885)
influxdata/docs-v2: content/kapacitor/v1.6/event_handlers/kafka.md

```bash
kapacitor define-topic-handler kafka_cpu_handler.yaml
```

### Using SASL with Kapacitor

> Kapacitor can be configured to use SASL if you want to use methods of authentication other than SSL. 

Reviewer: ```suggestion
To use an authentication method other than SSL, configure Kapacitor to use SASL. 
```

> An example would be using Kapacitor to authenticate directly against Kafka with a username/password. 

Reviewer: For example, to use Kapacitor to authenticate with a username/password directly against Kafka, configure SASL.

> Multiple configuration options are available, but the most common usage is username and password as shown in the following example:

Reviewer: Do we want to call out the other available config options?

Author: by far the most common is username/password, Emrys suggested just using that one
